From 9c8280fd2ed41326d424458bcd9cbdc524a6b235 Mon Sep 17 00:00:00 2001
From: Santan Kumar <santan.kumar@nxp.com>
Date: Thu, 22 Jun 2017 13:04:00 +0530
Subject: [PATCH 037/767] arm64: dts: ls2088ardb: Update nodes for QSPI

-As per board design, different QSPI flash is connected on
  boards, hence change QSPI flash node from s25fl256s1 to s25fs512ss in
  device tree.
-Enable fast-read support in QSPI node.

Signed-off-by: Santan Kumar <santan.kumar@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls208xa-rdb.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls208xa-rdb.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls208xa-rdb.dtsi
index e090816b1364..cbea5b6fd562 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls208xa-rdb.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls208xa-rdb.dtsi
@@ -146,10 +146,11 @@
 
 &qspi {
 	status = "okay";
-	flash0: s25fl256s1@0 {
+	flash0: s25fs512s@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "st,m25p80";
+		compatible = "spansion,m25p80";
+		m25p,fast-read;
 		spi-max-frequency = <20000000>;
 		reg = <0>;
 	};
-- 
2.17.0

